napari/components/_layer_slicer.py

```diff
@@ -102,7 +142,7 @@ def slice_layers_async(
         # when we want to perform sync slicing anyway.
         requests = {}
         for layer in layers:
-            if isinstance(layer, _AsyncSliceable):
+            if isinstance(layer, _AsyncSliceable) and not self._force_sync:
```

Comment:
I know that it should be pointed out earlier, but for me all async tasks should be started before the sync one.

Reply:
Do you mean you'd like to collect all the sync tasks in this loop (without calling `slice_dims`). Then one this loop is complete, and all async tasks have been submitted, we loop over the sync tasks and call `slice_dims` on each?

Reply:
Yes.